test/fixtures/dev-server/default-config.js

```diff
@@ -9,12 +9,9 @@ module.exports = {
   entry: resolve(__dirname, './foo.js'),
   devServer: {
     client: {
+      transport: 'sockjs',
```

Comment:
@snitin315 Again, here you do the same problem as do other developers, `client.transport` for only client, it is not create sockjs server, we need search better name for this, you should not set `sockjs`/`ws` server using this options, it is only if you want create custom web socket **client**

Reply:
I see we need configure `webSocketServer`  here.

Reply:
Yep, I am working on it, most of tests are good, anyway we need refactor many tests, I will create roadmap for rc